src/index.js

```diff
@@ -17,6 +19,34 @@ if (process.argv.length === 2) {
   } else if (input.includes("-h") || input.includes("--help")) {
     utils.helpManual();
     process.exit(0);
+  } else if (
+    (input.includes("-c") || input.includes("--config")) &&
+    input.length === 3
+  ) {
+    let filePath = "";
+    if (input[0] === "-c" || input[0] === "--config") {
+      filePath = input[2];
+      if (input[1].includes(".toml")) {
+        try {
+          fs.createReadStream(input[1], "utf8").pipe(
+            concat(function (data) {
+              var parsed = toml.parse(data);
+              utils.determinePath([filePath], parsed.output);
+              process.exit(0);
+            })
+          );
+        } catch (err) {
```

Comment:
When I input a config file that doesn't exist, such as: `TILerator -c confi.toml ./examples/til_Sample.txt`, it looks like the try catch block isn't catching this error. I believe when using .pipe() with Node, it doesn't forward the error so you need to handle the error before that.
Something like this may work, but a bit more changes is needed to show a proper message to the console and to exit the program.
```
fs.createReadStream(input[1], "utf8").on('error', err => console.log(err))
    .pipe(....
```
If this does work, please confirm if the try catch block is needed for any other possible errors, and if not, you can remove it.

Reply:
Added .on to createReadStream instead of try/catch blocks which doesn't really catch an error, and I removed it.